CHANGELOG.md

```diff
@@ -2,3 +2,5 @@
 # existing entries.
 # Use this layout for new entries:
 # <Area>: <Details about the change> [reference thread / issue]
+
+* Profile: Context menu entry to manually split a dive
```

Comment:
we should add entries on top of the file (above the note).

Reply:
Yes, please. Even with that the merge sometimes fails strangely enough, but at least chances are better.

Reply:
Fixed in updated version.